### ISSUE TITLE
Blocked phone numbers are not displayed

### ISSUE THREAD

**Issue by jean-io** [signalapp/Signal-iOS#4914]
Oh yes! Another free iPhone or MacBook from Amazon... I am so lucky these days.
Unless it's a spam 😱

![image](https://user-images.githubusercontent.com/4070933/110749222-1e6ee000-8241-11eb-8104-2e16725947a0.png)

The thing is: we are receiving a lot of spams these days (at least in France). This is my fifth Amazon's spam. And when I visit blocked page in my Signal app, I have the following screen:

![image](https://user-images.githubusercontent.com/4070933/110749597-a05f0900-8241-11eb-80f9-8951592e4bfb.png)

**The issue: blocked phone numbers should be displayed next to their attached name.** If not users like me thinks that other people are blocked by there names (which can be a useful feature).

- **Device**: iPhone X
- **iOS version**: 14.4
- **Signal version:** 5.7.0.19

**Comment by sergelenks** [comment]
Leave this one to me.

**Comment by sergelenks** [comment]
It should reference other strings and if different, show up as string digits for more detail.

**Comment by sergelenks** [comment]
That still doesn't solve the underlying issue that multiple amazon numbers are spamming, if the guess is correct.

**Comment by sergelenks** [comment]
Maybe it could also be a memory/thread issue for the block numbers feature.

**Comment by jean-io** [comment]
This issue is still present in latest releases. There is no activity because no one is working on it.

**Comment by jean-io** [comment]
Yes issue still relevant. It is just missing someone to take a look at it.

**Comment by NiklasBr** [comment]
Relevant.

**Comment by jean-io** [comment]
Relevant

**Comment by NiklasBr** [comment]
I can't see that anything has happened. The bot should take it easy.
